src/pages/ContactUs.jsx

```diff
@@ -0,0 +1,205 @@
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import { useRef } from 'react';
+
+import contactus from '../assets/contactus.svg';
+import sendEmail from '../utils/utils';
+
+function Contactus() {
+  const modelRef = useRef();
+
+  return (
+    <section className="text-gray-600 body-font relative">
+      <h1 className="text-gray-400 text-5xl p-5 font-medium">CONTACT US</h1>
+      <h2 className="text-3xl font-medium drop-shadow-md px-10">
+        Please get in touch and our expert team will answer all your questions.
+      </h2>
+      <div className="px-5 py-2 mx-auto flex sm:flex-nowrap flex-wrap flex-col md:flex-row items-center">
+        <div className="lg:w-2/3 md:w-1/2 w-full">
+          <img src={contactus} alt="Contact Us" />
+        </div>
+        <div className="rounded lg:w-1/3 md:w-1/2 bg-white flex flex-col md:ml-auto w-full md:py-8 mt-8 md:mt-0 justify-center p-4">
+          <Formik
+            initialValues={{ name: '', email: '', message: '' }}
+            validate={(values) => {
+              const errors = {};
+              if (values.email === '') {
+                errors.email = 'Email is required';
+              } else if (!/^[^\s@]+@[^\s@]+\.[^\s@]{2,}$/i.test(values.email)) {
+                errors.email = 'Invalid email address format';
+              }
+              if (values.name === '') {
+                errors.name = 'Name is required';
+              }
+              if (values.message === '') {
+                errors.message =
+                  'Please add a message at least one character long';
+              }
+              return errors;
+            }}
+            onSubmit={(values) => {
+              sendEmail(values, modelRef);
+            }}
+          >
+            <Form>
+              <div className="mb-4 text-left">
+                <Field name="name">
+                  {({ field }) => (
+                    <label
+                      htmlFor="name"
+                      className="uppercase text-sm text-gray-500 font-bold"
+                    >
+                      Full Name
+                      <input
+                        className="w-full bg-gray-200 text-gray-900 mt-2 p-3 rounded-lg focus:outline-none focus:shadow-outline"
+                        {...field}
```

Comment:
These gives out eslint warnings like no-props-spreading. You can see the warnings in your console when you npm start. Can you find a workaround about it ?

Reply:
I solved those errors